docs/tasks/administer-cluster/ip-masq-agent.md

```diff
@@ -0,0 +1,97 @@
+*   **masqLinkLocal:** A boolean (true / false) which indicates whether to masquerade traffic to the link local prefix 169.254.0.0/16. False by default.
+*   **resyncInterval: **An interval at which the agent attempts to reload config from disk. e.g. '30s' where 's' is seconds, 'ms' is milliseconds etc...
+
+Traffic to 10.0.0.0/8, 172.16.0.0/12 and 192.168.0.0/16) ranges will NOT be masqueraded. Any other traffic (assumed to be internet) will be masqueraded.  An example of a local destination from a pod could be its Node's IP address as well as another node's address or one of the IP addresses in Cluster's IP range.   Any other traffic will be masqueraded by default.  The below entries show the default set of rules that are applied by the ip-masq-agent:
+
+```
+iptables -t nat -L IP-MASQ-AGENT
+RETURN     all  --  anywhere             169.254.0.0/16       /* ip-masq-agent: cluster-local traffic should not be subject to MASQUERADE */ ADDRTYPE match dst-type !LOCAL
+RETURN     all  --  anywhere             10.0.0.0/8           /* ip-masq-agent: cluster-local traffic should not be subject to MASQUERADE */ ADDRTYPE match dst-type !LOCAL
+RETURN     all  --  anywhere             172.16.0.0/12        /* ip-masq-agent: cluster-local traffic should not be subject to MASQUERADE */ ADDRTYPE match dst-type !LOCAL
+RETURN     all  --  anywhere             192.168.0.0/16       /* ip-masq-agent: cluster-local traffic should not be subject to MASQUERADE */ ADDRTYPE match dst-type !LOCAL
+MASQUERADE  all  --  anywhere             anywhere             /* ip-masq-agent: outbound traffic should be subject to MASQUERADE (this match must come after cluster-local CIDR matches) */ ADDRTYPE match dst-type !LOCAL
+
+```
+
+By default in GCE/GKE starting with Kubernetes version 1.7.0 the ip-masq-agent will run in your cluster.  If you are running in another environment you can add the ip-masq-agent [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to your cluster:
+{% endcapture %}
+
+{% capture steps %}
+`
+kubectl create -f https://github.com/kubernetes-incubator/ip-masq-agent/blob/master/ip-masq-agent.yaml
+`
+More information can be found in the ip-masq-agent documentation [here](https://github.com/kubernetes-incubator/ip-masq-agent)
+
+In most cases the default set of rules should be sufficient, however, if this is not the case for your cluster, you can create and apply a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configmap/) to customize the IP ranges that are affected.  For example, to allow only 10.0.0.0/8 to be considered by the ip-masq-agent, you can create the following [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configmap/) in a file called "config".  **Note:** It is important that the file is called config since, by default that will be used as the key for lookup by the ip-masq-agent:
```

Comment:
Add comma: "In most cases`,` the default..."

Use semicolon: "...should be sufficient`;` however, if..."

Use a relative path for the ConfigMap link: \[ConfigMap](/docs/tasks/configure-pod-container/configmap/)

Put the Note on a separate line.

Add comma: "...since, by default`,` that will..."